src/AWS.Deploy.Orchestration/DisplayedResources/ElasticLoadBalancerResource.cs

```diff
@@ -0,0 +1,29 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using AWS.Deploy.Orchestration.Data;
+
+namespace AWS.Deploy.Orchestration.DisplayedResources
+{
+    public class ElasticLoadBalancerResource : IDisplayedResourceCommand
+    {
+        private readonly IAWSResourceQueryer _awsResourceQueryer;
+
+        public ElasticLoadBalancerResource(IAWSResourceQueryer awsResourceQueryer)
+        {
+            _awsResourceQueryer = awsResourceQueryer;
+        }
+
+        public async Task<Dictionary<string, string>> Execute(string resourceId)
+        {
+            var loadBalancer = await _awsResourceQueryer.DescribeElasticLoadBalancer(resourceId);
```

Comment:
I think we should put the display resources in the recipe on the load balancer listener. Then we can describe that and get the protocol it is configured (http or https) instead of always assuming it is http. Either that or we could from here list the listeners and if there is an https listener prefer that protocol over http.

Reply:
Fixed